src/index.jsx

```diff
@@ -15,10 +15,10 @@ import './styles/styles.css';
 
 import 'antd/dist/antd.less';
 import { NotFoundPage } from './components/pages/NotFound';
-import { ExampleListPage } from './components/pages/ExampleList';
+import { ExampleListPage } from './components/UNUSED/ExampleList';
```

Comment:
It's a good idea to leave the unused components available just in case for another week, at least until we get the framework of the site up and solidified, but when we go to delete the unused components we need to remember to remove all the react router Routes and Links, as well as these imported components.

Reply:
@attaris978 by removing this unused component there could be potential for a significant cascading refactor?